src/components/navigation/navigation.tsx

```diff
@@ -268,6 +268,7 @@ const Navigation: React.FC<NavigationProps> = ({
                 <LanguageSwitchWrapper
                   translation={translation}
                   className={'language-switch'}
+                  fromNavigation
```

Comment:
kind of awkward to have the language selection in the navigation only when it's triggered by the navigation and not when I scroll down (look at the footer, where the language select is appearin/disappearing when I click). Is this the intention of `fromNavigation` ? (I couldn't find anything else related that would match this behavior). Thanks for your clarification.

https://user-images.githubusercontent.com/1701755/140721057-fab95474-e3ad-437f-8f4b-c173cceec8ae.mp4

Reply:
`fromNavigation` only ensures that the LanguageSwitch gets the right color. The distinction of whether the NavigationSwitch is displayed or not is made in the actual navigation component (`showLanguageSwitch`) and depends on whether the navigation is displayed as a footer or as a flyout. 
I agree with you that the behavior is a bit unexpected. I can talk to Eric and ask him if it would make sense to always show the language switch in the navigation.

Reply:
If we don't find a solution for this I'm happy to tackle this later so you can move forward with the PR 👍